Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -3880,6 +3881,47 @@ UserSelect RenderStyle::effectiveUserSelect() const
     return value;
 }
 
+const Vector<Ref<ScrollTimeline>>& RenderStyle::scrollTimelines() const
+{
+    return m_nonInheritedData->rareData->scrollTimelines;
+}
+
+const Vector<ScrollAxis>& RenderStyle::scrollTimelineAxis() const
+{
+    return m_nonInheritedData->rareData->scrollTimelineAxes;
+}
+
+const Vector<AtomString>& RenderStyle::scrollTimelineName() const
+{
+    return m_nonInheritedData->rareData->scrollTimelineNames;
+}
+
+void RenderStyle::setScrollTimelineAxis(const Vector<ScrollAxis>& axes)
+{
+    m_nonInheritedData.access().rareData.access().scrollTimelineAxes = axes;
+}
+
+void RenderStyle::setScrollTimelineName(const Vector<AtomString>& names)
+{
+    m_nonInheritedData.access().rareData.access().scrollTimelineNames = names;
+}
```

Comment:
Can we use the plural names on RenderStyle? You can use `"name-for-methods"` in `CSSProperties.json` if you want to link them to each other.

Reply:
Also should these methods be in `RenderStyleInlines.h` and `RenderStyleSetters.h` respectively? (No strong opinion here)

Reply:
Yeah, I think the plural name would be nicer, I'll set this up. I'll also look at `RenderStyleSetters.h`. Not sure what qualifies to be in `RenderStyleInlines.h`.